silk/management/commands/silk_request_garbage_collect.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+
+import silk.models
+from silk.config import SilkyConfig
+
+
+class Command(BaseCommand):
+    help = "Triggers silk's request garbage collect."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-m",
+            "--max-requests",
+            default=SilkyConfig().SILKY_MAX_RECORDED_REQUESTS,
+            type=int,
+            help="Maximum number of requests to keep after garbage collection.",
+        )
+
+    def handle(self, *args, **options):
+        if "max_requests" in options:
+            max_requests = options["max_requests"]
+            SilkyConfig().SILKY_MAX_RECORDED_REQUESTS = max_requests
```

Comment:
Changing `SilkyConfig` in flight is kind of tricky because these changes aren't propagated back into the original django project's `settings.py`, possibly making it hard to debug issues.  Unfortunately, I think this is the only way to do so because `SilkyConfig` is a singleton and instantiated only once before this code will run.